PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalLauncher.kt

```diff
@@ -67,10 +67,18 @@ class PayPalLauncher internal constructor(
         }
 
         val options = paymentAuthRequest.requestParams.browserSwitchOptions
-        if (options == null) {
-            val error = BraintreeException("BrowserSwitchOptions is null")
-            return sendLaunchFailureEventAndReturn(error, isAppSwitch, analyticsEventParams)
-        }
+            ?: return buildLaunchFailureAndReturn(
+                "BrowserSwitchOptions is null",
+                isAppSwitch,
+                analyticsEventParams
+            )
+
+        options.url
+            ?: return buildLaunchFailureAndReturn(
+                "BrowserSwitchOptions URL is null",
+                isAppSwitch,
+                analyticsEventParams
+            )
```

Comment:
Suggestion - named arguments may help with readability here, especially for the first argument.

```suggestion
            ?: return buildLaunchFailureAndReturn(
                errorMessage = "BrowserSwitchOptions is null",
                isAppSwitch = isAppSwitch,
                analyticsEventParams = analyticsEventParams
            )

        options.url
            ?: return buildLaunchFailureAndReturn(
                errorMessage = "BrowserSwitchOptions URL is null",
                isAppSwitch = isAppSwitch,
                analyticsEventParams = analyticsEventParams
            )
```